sharepoint/sharepoint-ps/sharepoint-online/Get-SPODataAccessGovernanceInsight.md

```diff
@@ -53,7 +53,7 @@ Specifies the entity that could cause oversharing and hence tracked by these rep
 Type: ReportEntityEnum
 Parameter Sets: GetAllReportsParameterSet
 Aliases:
-Accepted values: SharingLinks_Anyone, SharingLinks_PeopleInYourOrg, SharingLinks_Guests, SensitivityLabelForFiles, EveryoneExceptExternalUsersAtSite, EveryoneExceptExternalUsersForItems, PermissionedUsers
+Accepted values: SharingLinks_Anyone, SharingLinks_PeopleInYourOrg, SharingLinks_Guests, SensitivityLabelForFiles, EveryoneExceptExternalUsersAtSite, EveryoneExceptExternalUsersForItems, PermissionedUsers, PermissionsReport (Preview)
```

Comment:
> (Preview) [](http://example.com/codeflow?start=223&length=10)

remove `(preview)` since it's not part of the accepted value -- it's an explanation

Reply:
applies to the other file as well